packages/rolldown/src/utils/compose-js-plugins.ts

```diff
@@ -271,7 +255,7 @@ function isComposablePlugin(plugin: RolldownPlugin): plugin is Plugin {
     return false
   }
 
-  if (Object.keys(plugin).some((k) => unsupportedHooks.has(k))) {
+  if (R.keys(plugin).some((hookName) => isUnsupportedHooks(hookName))) {
```

Comment:
`isComposablePlugin` now shared the same logic on detecting unsupported plugin with `createComposedPlugin`